package.json

```diff
@@ -82,7 +83,7 @@
     "dependencies": {
         "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#6.1.1",
         "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#4.0.1",
-        "@duckduckgo/ddg2dnr": "github:duckduckgo/ddg2dnr#0.3.0",
+        "@duckduckgo/ddg2dnr": "github:duckduckgo/ddg2dnr#f6f1d9c968d645641489dec38ce11ff843e59141",
```

Comment:
Don't forget to update after we do a new dnr version.

Reply:
I've [cut a release (0.3.1)](https://github.com/duckduckgo/ddg2dnr/commit/c11effd859087313a80b382623648b3a612a98c6) in case that helps.